tests/browser/playwright.config.ts

```diff
@@ -84,7 +84,7 @@ const config: PlaywrightTestConfig = {
   /* Opt out of parallel tests on CI. */
   workers: isCI ? 1 : undefined,
   /* Reporter to use. See https://playwright.dev/docs/test-reporters */
-  reporter: [['html', { outputFolder: 'playwright-report' }]],
+  reporter: [['html', { open: 'never' }]],
```

Comment:
Why the change here? My understanding is that this would have outputted a browseable html page to see the failures

Reply:
It still generates the html it just doesn't try to open it automatically which I found to be annoying. The notes it gives after running are `npx view-report` to view the report if you want to.

Reply:
It also may someday cause problems in CI

Reply:
If we don't specify the `outputFolder`, will it default to some folder name that is relative to the tests?

Reply:
It goes to the same place, the output folder we had specified was same as the default.